project_4/index.html

```diff
@@ -0,0 +1,384 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,700|Raleway:400,700" rel="stylesheet">
+    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+    <title>Sustainability by Design Conference</title>
+</head>
+
+<!-- I wrote most of the text throughout the page, I cited outside sources wherever I used them -->
+
+<body>
+    <!-- empty div for the header in the naviation to go to the top of the page -->
+    <div id="pagetitle"></div>
```

Comment:
You could just add this `id` to your `body` element and eliminate the empty `div` 🙌🏻